### PR TITLE
Improvements for Issue Triage

### PR DESCRIPTION
**Pull request by jon-funk**
- improves issues / docstrings / titles across deployment + namespace tasksets
- adds 2 new tasks to namespace triage: event anomaly detection, service log aggregation and inspection
- dedupes tasks in artifactory triage - covered by statefulset triage

### CODE REVIEW

**Comment by jon-funk** [comment]
Note the issue max length bump to 2048, this was done to allow for enough room to fit useful log results in the issue so that personas can use them to infer what to check next.

File: codebundles/k8s-artifactory-health/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Configuration
+The TaskSet requires initialization to import necessary secrets, services, and user variables. The following variables should be set:
+
+- `kubeconfig`: The kubeconfig secret containing access info for the cluster.
+- `kubectl`: The location service used to interpret shell commands. Default value is `kubectl-service.shared`.
+- `KUBERNETES_DISTRIBUTION_BINARY`: Which binary to use for Kubernetes CLI commands. Default value is `kubectl`.
+- `CONTEXT`: The Kubernetes context to operate within.
+- `NAMESPACE`: The name of the namespace to search. Leave it blank to search in all namespaces.
+- `DEPLOYMENT_NAME`: The name of the deployment running Redis
```
Comment (stewartshea) [inline codebundles/k8s-artifactory-health/README.md:16]: Was this copied from another doc? I didn't see a redis deployment tied to artifactory, at least not discovered by rw-local

File: codebundles/k8s-namespace-healthcheck/runbook.robot [hunk 2]
```diff
@@ -42,6 +42,24 @@ Suite Initialization
...
+    ...    pattern=\w*
+    ...    example=(Error: 13|Error: 14)
+    ...    default=(Error:)
+    ${SERVICE_EXCLUDE_PATTERN}=    RW.Core.Import User Variable    SERVICE_EXCLUDE_PATTERN
+    ...    type=string
+    ...    description=Pattern used to exclude entries from log results when searching in service logs.
+    ...    pattern=\w*
+    ...    example=(node_modules|opentelemetry)
+    ...    default=(node_modules|opentelemetry)
+    ${ANOMALY_THRESHOLD}=    RW.Core.Import User Variable    ANOMALY_THRESHOLD
```
Comment (stewartshea) [inline codebundles/k8s-namespace-healthcheck/runbook.robot:57]: @jon-funk I was thinking about an x count over y minutes. I think k8s keeps the events for an hour, but a more normal pattern would be something like seeing a pull event 5 times in 5 minutes. Do you think something like that is possible?

**Review by stewartshea — APPROVED** [review]

File: codebundles/k8s-artifactory-health/README.md (hunk 1 above)
Comment (jon-funk) [reply codebundles/k8s-artifactory-health/README.md]: correct, I copy pasta'd too hard

File: codebundles/k8s-namespace-healthcheck/runbook.robot (hunk 2 above)
Comment (jon-funk) [reply codebundles/k8s-namespace-healthcheck/runbook.robot:57]: Example of new rate calc in raised issue:

`Event(s) Per Minute:5.470709500085896 Count:8492 Object:online-boutique/Pod/crashpod-7d4984c5d6-8fzzg Reason:NotTriggerScaleUp Message:pod didn't trigger scale-up: 3 pod has unbound immediate PersistentVolumeClaims`

**Comment by stewartshea** [comment]
Nice thanks!

**Pull request merged**